src/modes.cpp

```diff
@@ -3,6 +3,12 @@
 #include "logger.hpp"
 #include "utilities.hpp"
 
+#ifdef PRECICE_VERSION
+#if PRECICE_VERSION_GREATER_EQUAL(2, 5, 0)
+#define ASTE_NN_GRADIENT_MAPPING_AND_TETRA
+#endif
+#endif
```

Comment:
```suggestion
#ifndef PRECICE_VERSION_GREATER_EQUAL
   // compatibility with older versions
   #define PRECICE_VERSION_GREATER_EQUAL(x,y,z) FALSE
#endif
```
Please put this in one place into a header file so that the macro is available everywhere.

Reply:
It moved to `modes.hpp`